aixweather/imports/DWD.py

```diff
@@ -208,6 +208,7 @@ def import_meta_DWD_historical(station: str) -> utils_import.MetaData:
     meta.station_exists_since = station_data["BEGINN"]
     meta.station_exists_until = station_data["ENDE"]
     meta.input_source = "DWD Historical"
+    meta.timezone = 1  # Always in Germany
```

Comment:
Bin mir hier nicht sicher. 1 ist ja nur winterzeit. Die Daten die man beim DWD lädt, waren (glaube ich) UTC. 
Erklär hier mal die Wirkung der Zeile, ansonsten müsste die Timezone auch in dem "dict" stehen. Ist aber alles bisschen was her.